.github/workflows/ci.yml

```diff
@@ -12,6 +12,14 @@ jobs:
     name: Lint JSON
     runs-on: ubuntu-latest
     steps:
-      - run: sudo apt-get install -y python3-demjson
-      - uses: actions/checkout@v2
-      - run: jsonlint --strict ./directory.json
+      - run: sudo apt install -y python3-demjson
```

Comment:
`apt` prints a warning, that you shouldn't use it in scripts because of the non-stable output:

<img width="715" height="130" alt="Image" src="https://github.com/user-attachments/assets/28cd1054-69c6-40f2-b76e-feb064139b1d" />

So what's wrong with using `apt-get`?